src/lib/actionrunner.js

```diff
@@ -167,6 +167,7 @@ class OpenwhiskActionRunner {
         this.containerId = await this._docker(
             `run -d
                 --rm
+                --security-opt seccomp=unconfined
```

Comment:
If we use an env variable to set seccomp, we would not need to change the code once we have a good security policy. Wdyt? (Default if nothing is set for now would be `unconfined`).

Reply:
Ultimately we need a better security policy that makes sense for general use, so I'd expect this line to be removed sooner than later (I hope!)